### PR TITLE
Add Account Management and SAML documentation pages

### PR DESCRIPTION
**Pull request by lfittl**
In addition, move existing Permissions and Roles page to the new
"accounts" namespace in the documentation, with a redirect being added
in the web repository.

### CODE REVIEW

File: accounts/sso/okta/index.mdx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+https://[OKTA_ORG_NAME].okta.com/app/[OKTA_ENTITY_ID]/sso/saml/metadata
+```
+
+## Register the Okta application with pganalyze
+
+Specify the **Identity Provider metadata** URL in pganalyze on the **Integrations** settings page:
+
+![Screenshot of pganalyze SAML integration page showing Okta IDP metadata URL](pganalyze_okta_idp_metadata_url.png)
+
+Once you submit the Metadata URL, pganalyze will verify the configuration, and either provide an error,
```
Comment (uhoh-itsmaciek) [inline accounts/sso/okta/index.mdx:74]: Are there any troubleshooting steps we can offer here if users get an error? Or is this likely to be obvious?

File: accounts/sso/saml2.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+To tell pganalyze which IdP you trust, you need to retrieve the metadata URL from the IdP, and then configure
+it on the pganalyze **Integrations** settings page:
+
+![Screenshot of pganalyze SAML integration form for the IdP metadata URL](sso_idp_metadata_form.png)
+
+Note that this is the only method of configuration supported, at this point you cannot specify Service Provider (SP) settings
+directly. Please reach out to [pganalyze Support](mailto:support@pganalyze.com) for alternate setup methods.
+
+Once you submit the Metadata URL, pganalyze will verify the configuration, and either provide an error,
```
Comment (uhoh-itsmaciek) [inline accounts/sso/saml2.mdx:48]: Same as above re: errors.

File: accounts/sso/saml2.mdx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+![Screenshot of successful pganalyze SAML integration](sso_success.png)
+
+SSO setup is now complete. Going forward your team members can now sign in to pganalyze through your Identity Provider (IdP).
+
+[Read more about how to migrate users and general Single Sign-On functionality](/docs/accounts/sso).
+
+## Reference: pganalyze SAML IdP settings template
+
+When not copying the values directly from the pganalyze app (recommended), you can construct the IdP settings based on this template.
```
Comment (uhoh-itsmaciek) [inline accounts/sso/saml2.mdx:59]: It's not clear what mechanism is recommended ("copying" or "not copying") with this wording. Maybe reword slightly, something like
> We recommend copying the values directly from the pganalyze app, but you can alternately construct the IdP settings based on this template.

?

File: accounts/sso/saml2.mdx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+* Audience URI: `app.pganalyze.com`
+
+The format for pganalyze Enterprise Server installs is similar, but based on your chosen domain name:
+
+* ACS URL: `https://[DOMAIN_NAME]/users/auth/saml/[ORGANIZATION_SHORT_NAME]/callback`
+* Metadata URL: `https://[DOMAIN_NAME]/users/auth/saml/[ORGANIZATION_SHORT_NAME]/metadata`
+* Audience URI: `[DOMAIN_NAME]`
+
+Replace `[ORGANIZATION_SHORT_NAME]` with your organization's short name
+(e.g. used in `/organizations/[ORGANIZATION_SHORT_NAME]` URLs).
```
Comment (uhoh-itsmaciek) [inline accounts/sso/saml2.mdx:74]: Maybe move this up above the Enterprise instructions? Users may ignore that part since it's not applicable, and it's not clear where to pick up again. If this is above the Enterprise instructions, they can just ignore the rest of the doc.

**Review by uhoh-itsmaciek — APPROVED** [review]
Excellent.

File: accounts/sso/okta/index.mdx (hunk 1 above)
Comment (lfittl) [reply accounts/sso/okta/index.mdx:74]: I'm not sure - I think it might be best if we tweak the docs based on error reports - the general error pattern here is `Error retrieving metadata information`, but I'm not 100% on what the possible problems are (outside of the obvious HTTP errors, which I don't think would be encountered frequently).
